app/models/validations/sales/sale_information_validations.rb

```diff
@@ -147,15 +147,6 @@ def validate_staircasing_mortgage(record)
     end
   end
 
-  def validate_mortgage_used_and_stairbought(record)
-    return unless record.stairowned && record.mortgageused
-
-    if !record.stairowned_100? && record.mortgageused == 3
-      record.errors.add :stairowned, I18n.t("validations.sale_information.stairowned.mortgageused_dont_know")
-      record.errors.add :mortgageused, I18n.t("validations.sale_information.stairowned.mortgageused_dont_know")
-    end
-  end
```

Comment:
we can see these validations are only triggered when mortgage used is 3, ie don't know, so I combined them into the method below